ckanext/switzerland/templates/admin/trash.html

```diff
@@ -0,0 +1,37 @@
+{% ckan_extends %}
+
+{% block primary_content_inner %}
+  {% set truncate = truncate or 180 %}
+  {% set truncate_title = truncate_title or 80 %}
```

Comment:
I have never seen that notation, can you explain to me what setting it to 180 or 80 does here?

Reply:
This is copied over from the CKAN template - I think it's to make the template more flexible. You can either pass in values for `truncate` and `truncate_title`, or use the defaults (180 and 80). In line 9, `truncate_title` is used as the length to which the title should be truncated.

Looking at it now, it seems that we never use the variable `truncate`, and so far as I know we never use any truncate-length besides the default anyway. This could be cleaned up if you think it's confusing like this.

Reply:
ah, thank you for the explanation :) I think using the reasoning that we override from CKAN where we need it, I would leave it like that. Unless we need it differently, of course, but probably makes it easier to read commit-wise ('copy template'-commit and then everything we need differently)

TLDR: Let's keep it, thank you for explaining this to me 😄